tests/dbus/test_dbus_client.cpp

```diff
@@ -263,14 +263,20 @@ void CheckNat64(ThreadApiDBus *aApi)
 
 void CheckEphemeralKey(ThreadApiDBus *aApi)
 {
-    bool enabled;
+    bool        enabled;
+    std::string ePSKc = '0';
```

Comment:
I think this may cause problem. Have you verified the issue still exists when you fixed this?